.release-please-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  ".": "2.3.2",
-  "packages/api-explorer": "0.9.32",
-  "packages/code-editor": "0.1.21",
-  "packages/extension-api-explorer": "22.4.2",
-  "packages/extension-sdk": "22.4.2",
-  "packages/extension-sdk-react": "22.4.2",
-  "packages/extension-utils": "0.1.9",
-  "packages/hackathon": "22.4.2",
-  "packages/run-it": "0.9.32",
-  "packages/sdk": "22.4.2",
-  "packages/sdk-codegen": "21.5.2",
-  "packages/sdk-codegen-scripts": "21.3.3",
+  ".": "2.4.0",
+  "packages/api-explorer": "0.9.33",
+  "packages/code-editor": "0.1.22",
+  "packages/extension-api-explorer": "22.6.0",
+  "packages/extension-sdk": "22.6.0",
+  "packages/extension-sdk-react": "22.6.0",
+  "packages/extension-utils": "0.1.10",
+  "packages/hackathon": "22.6.0",
+  "packages/run-it": "0.9.33",
+  "packages/sdk": "22.6.0",
+  "packages/sdk-codegen": "21.6.0",
+  "packages/sdk-codegen-scripts": "21.3.4",
   "packages/sdk-codegen-utils": "21.0.11",
-  "packages/sdk-node": "22.4.2",
-  "packages/sdk-rtl": "21.3.3",
-  "packages/wholly-sheet": "0.5.28",
-  "python": "22.4.0"
+  "packages/sdk-node": "22.6.0",
+  "packages/sdk-rtl": "21.3.4",
+  "packages/wholly-sheet": "0.5.29",
+  "python": "22.6.0"
```

Comment:
@jkaster @fabio-looker @jeremytchang these versions seem OK to me for the 22.6.0 release. Can you take a look and double check?

Reply:
I double checked all the auto-incremented versions for the all the public packages and they look good. For the explicitly set 22.6.0 versions, I am not sure what I would double-check, but it seems ok!